src/Components/SearchSports.jsx

```diff
@@ -0,0 +1,18 @@
+import { useContext } from "react";
+import { SearchKeywordContext } from "./SearchKeywordContext";
+import "./SearchSports.scss";
+
+function SearchSports () {
+  const { searchKeyword, setSearchKeyword } = useContext(SearchKeywordContext);
```

Comment:
If I'm not mistaken, you are not using `searchKeyword`.

Reply:
You are right.I removed it.